pkg/network/driver_interface.go

```diff
@@ -50,13 +53,27 @@ var (
 // DriverExpvar is the name of a top-level driver expvar returned from GetStats
 type DriverExpvar string
 
+// This is the type that an overlapped read returns -- the overlapped object, which must be passed back to the kernel after reading
+// followed by a predictably sized chunk of bytes
+type _readbuffer struct {
+	ol windows.Overlapped
+
+	// This is the MTU of IPv6, which effectively governs the maximum DNS packet size over IPv6
+	// see https://tools.ietf.org/id/draft-madi-dnsop-udp4dns-00.html
+	data [1280]byte
```

Comment:
> For example, with Ethernet, the maximum frame size is 1518 bytes, 18 bytes of which are overhead (header and frame check sequence), resulting in an MTU of 1500 bytes.

The MTU for IPv4 can be up to 1500. Should we increase this buffer size?

Reply:
I'm not exactly sure, but the difference in memory is negligible so done